### DISCUSSION TITLE
Bug? Font property not working in v0.28.0

### DISCUSSION THREAD

**Discussion by ArianYN** [sqlpage/SQLPage#586] [Q&A] [answered]
Hello, we just updated from v0.27.0 to v0.28.0, and suddenly the Font property of the "Shell" is no longer working
Here is an image of the comparison. (Top image is 0.27.0, and bottom is 0.28.0)
![FontNotWorking](https://github.com/user-attachments/assets/fdafa9e1-328e-4ba0-a8ec-3e4718d72093)

This is the Shell code. It is the exact same for both.
`SELECT 'shell' AS component,
    'Gateway Management' AS title,
    'tool' AS icon,
    16 as font_size,
    'Icons/makeenfavicon.ico' as favicon,
    'Poppins' as font`

Is this a bug?

**Comment by lovasoa** [comment]
Thank you for reporting this ! It had been mistakenly removed. I re-added the font property.

**Reply by ArianYN** [reply]
Okay, do we need to wait for the next release to get the property back?

**Reply by lovasoa** [reply, marked as answer]
Yes, this will be in v0.29. A pre-release is always available on docker hub as `lovasoa/sqlpage:main`.
Alternatively, if you need the fix in v0.28, you can manually copy [the fixed `shell.handlebars`](https://github.com/lovasoa/SQLpage/blob/main/sqlpage/templates/shell.handlebars) to `sqlpage/templates` in your installation.
